crossbeam-channel/src/select.rs

```diff
@@ -633,6 +636,28 @@ impl<'a> Select<'a> {
         Self {
             handles: Vec::with_capacity(4),
             next_index: 0,
+            biased: false,
+        }
+    }
+
+    /// Creates an empty list of channel operations with biased selection.
+    ///
+    /// When multiple handles are ready, this will select the operation with the lowest index.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use crossbeam_channel::Select;
+    ///
+    /// let mut sel = Select::with_bias();
+    ///
+    /// // The list of operations is empty, which means no operation can be selected.
+    /// assert!(sel.try_select().is_err());
+    /// ```
+    pub fn with_bias() -> Self {
+        Self {
+            biased: true,
+            ..Default::default()
```

Comment:
Why are you doing things differently here than `new`? (about `handles` field)

Reply:
If you e.g. decide that the handles field should be initialized differently when a new `Select` is created, it would only need to be edited in the `new` function and not in both. `new_biased` would use the same values for all fields except for the `biased` field.

I can change it if you want :)

Reply:
Ah, sorry, I missed Default::default just calls Select::new.